src/main/java/com/github/wovnio/wovnjava/Settings.java

```diff
@@ -96,6 +100,9 @@
         this.outboundProxyPort = reader.getIntParameter("outboundProxyPort");
 
         this.encoding = stringOrDefault(reader.getStringParameter("encoding"), "");
+        this.fixedHost = stringOrDefault(reader.getStringParameter("fixedHost"), "");
+        this.fixedScheme = stringOrDefault(reader.getStringParameter("fixedScheme"), "");
+        this.fixedPort = stringOrDefault(reader.getStringParameter("fixedPort"), "");
```

Comment:
use `positiveIntOrDefault`. Also I want to validate that if you specify one of these settings, you must specify all of them, even in the port number.

Reply:
Oh. Will add that.